### DISCUSSION TITLE
Can you provide a few simple examples?

### DISCUSSION THREAD

**Discussion by gedo4547** [nxrighthere/UnrealCLR#6] [General]
Hello, I've used auto-compilation. But after that, I don't know what to do.
I think I need some simple examples (such as UnrealCLR-based Hello World) to get me off a fast start.

**Comment by schotime** [comment]
Have a look at the Tests project. If you copy that into your Content as the instructions say in the manual step you will see how its being used in practice.

**Comment by nxrighthere** [comment]
Just as Adam said, you have to install the content of the tests and let the installation program launch the compilation of assemblies. See [this](https://github.com/nxrighthere/UnrealCLR#running) section for further steps.
